lib/grammars.coffee

```diff
@@ -137,7 +137,11 @@ module.exports =
   Go:
     "File Based":
       command: "go"
-      args: (context) -> ['run', context.filepath]
+        args: (context) ->
+          if context.filepath.match(/_test.go/)
+            ['test', '']
```

Comment:
Absolutely. go test looks for _test.go files in the same directory and execute the Testxxx functions in them.